_schemaV2.graphql

```diff
@@ -16890,6 +16890,7 @@ type Query {
     limit: Int
     offset: Int
     sort: DiscoverArtworksSort
+    useRelatedArtworks: Boolean = false
```

Comment:
I don't quite follow how this will be used. If the client will specify it, doesn't that remove our ability to swap data sources independently?

Reply:
I think it's helpful at the research phase to have both options if we want to compare model evaluations side-by-side. Otherwise, we could just use the environment variable.